packages/components/src/components/tooltip/tooltip.ts

```diff
@@ -161,10 +154,14 @@ export default class SdTooltip extends SolidElement {
   }
 
   private handleKeyDown(event: KeyboardEvent) {
-    // Pressing escape when the target element has focus should dismiss the tooltip
-    if (this.open && event.key === 'Escape') {
+    // Pressing escape, space or enter when the tooltip is open should dismiss the tooltip
+    if (this.open && (event.key === 'Escape' || event.key === 'Enter' || event.key === ' ')) {
+      event.preventDefault();
       event.stopPropagation();
       this.hide();
+    } else if ((event.key === 'Enter' || event.key === ' ') && !this.open) {
+      event.preventDefault();
+      this.show();
     }
   }
```

Comment:
Why did you have to implement this?

Reply:
Yeah I see now that wrapping the icon in a button makes this unnecessary.